src/org/traccar/database/DeviceManager.java

```diff
@@ -43,14 +41,12 @@
     private final DataManager dataManager;
     private final long dataRefreshDelay;
 
-    private final ReadWriteLock devicesLock = new ReentrantReadWriteLock();
-    private final Map<Long, Device> devicesById = new HashMap<>();
-    private final Map<String, Device> devicesByUniqueId = new HashMap<>();
-    private long devicesLastUpdate;
+    private Map<Long, Device> devicesById;
+    private Map<String, Device> devicesByUniqueId;
+    private AtomicLong devicesLastUpdate = new AtomicLong(0);
```

Comment:
You don't need to explicitly specify 0. Default constructor initializes value to 0.